src/api/app/views/webui/user/show.html.haml

```diff
@@ -0,0 +1,170 @@
+- @pagetitle = "Home of #{@displayed_user}"
+- @layouttype = 'custom'
+- @crumb_list = [@pagetitle]
+/ user
+#userinfo.grid_4.alpha.box.box-shadow
+  / avatar
+  = user_icon(@displayed_user, 200, 'home-avatar') if @displayed_user.email
+  / info
+  %h1
+    %span#home-realname
+      - if @displayed_user.realname.present?
+        = @displayed_user.realname
+        %br/
+    %span#home-username= @displayed_user.login if @displayed_user.login
+  / mail
+  - unless User.current.is_nobody?
+    %p
+      = mail_to(@displayed_user.email, sprite_tag('email', title: 'Send mail to user'), title: 'Send mail to user')
+      = mail_to(@displayed_user.email, nil, title: 'Send mail to user')
+  / groups
+  - gs = @displayed_user.groups.pluck(:title)
+  - unless gs.empty?
+    %p Member of the group
+    %ul
+      - gs.each do |group|
+        %li= link_to(group, controller: 'groups', action: 'show', title: group)
+  / roles
+  - grs = @displayed_user.roles.global.pluck(:title)
+  - if grs.present?
+    %p Has the role
+    %ul
+      - grs.each do |role|
+        %li= role
+  / edit
+  - if @is_displayed_user
+    %p
+      - if CONFIG['proxy_auth_mode'] == :on
+        - unless CONFIG['proxy_auth_account_page'].blank?
+          = link_to sprited_text('user_edit', 'Edit your account'), "#{CONFIG['proxy_auth_account_page']}"
+          %br/
+      - else
+        = link_to(sprited_text('user_edit', 'Edit your account'), { controller: 'user', action: 'save_dialog', user: User.current }, {id: 'save_dialog', remote: true})
+        %br/
+      - if @configuration['change_password'] && CONFIG['proxy_auth_mode'] != :on
+        = link_to(sprited_text('key', 'Change your password'), { controller: 'user', action: 'password_dialog', user: User.current }, {id: 'password_dialog', remote: true})
+        %br/
+      = link_to(sprited_text('email', 'Change your notifications'), user_notifications_path)
+      %br/
+.grid_12.omega.box.box-shadow
+  #involved
+    .box-header.header-tabs
+      %ul
+        %li
+          %a{href: "#ipackages", title: "Packages that you are involved with"} Involved Packages
+        %li
+          %a{href: "#iprojects", title: "Projects that you are involved with"} Involved Projects
+        %li
+          %a{href: "#iowned", title: "Projects/Packages where you are responsible for the maintenance"} Owned Project/Packages
+    #ipackages.tab
+      - if @ipackages.length > 0
+        #ipackages_wrapper{ "data-url": url_for(controller: 'package', action: 'show', project: 'REPLACEPRJ', package: 'REPLACEPKG') }
+          - content_for :ready_function do
+            var ipackages = [  #{escape_nested_list(@ipackages)}];
+            renderPackagesProjectsTable({packages: ipackages, length: 10, name: 'ipackages_wrapper'});
+      - else
+        %p
+          %i
+            = @displayed_user.login
+            is not involved in any packages
+    #iprojects.tab
```

Comment:
Class should be before id: https://github.com/brigade/haml-lint/blob/master/lib/haml_lint/linter/README.md#classesbeforeids